ui/de-lib/src/main/java/org/iplantc/de/apps/widgets/client/view/AppLaunchViewImpl.java

```diff
@@ -42,15 +43,22 @@
     private final EditorDriver editorDriver = GWT.create(EditorDriver.class);
 
     private final LaunchAnalysisView law;
+    private CustomMask customMask;
```

Comment:
The `CustomMask` and its appearance files are all basically so I can mask the view and NOT have the loading cursor.  I thought having the loading cursor might confuse users into thinking they might still be able to use the app if they wait for the "load" to finish.